### DISCUSSION TITLE
no test folder for "Zksync Devops" lecture

### DISCUSSION THREAD

**Discussion by khandelwalmoksh** [Cyfrin/foundry-full-course-cu#2816] [Q&A] [answered]
Not able to locate test folder in github repo. Lecture: "Zksync Devops" from Foundry Fund Me> Section 2> number 24. Zksync Devops

**Comment by EngrPips** [comment]
Hello @khandelwalmoksh, Did Patrick write a test file for `ZKSync Devops` in the course?

**Reply by khandelwalmoksh** [reply]
yes, he copied code from it. Actually its a folder, sorry

**Reply by EngrPips** [reply]
Please send a link to the lesson where he copied it

**Reply by khandelwalmoksh** [reply]
https://updraft.cyfrin.io/courses/foundry/foundry-fund-me/zksync-devops

**Reply by EngrPips** [reply, marked as answer]
Here https://github.com/Cyfrin/foundry-fund-me-cu/blob/main/test/unit/ZkSyncDevOps.t.sol is the link to the test file you are asking.

**Reply by khandelwalmoksh** [reply]
thankyou, can you tell me how you found this

**Reply by EngrPips** [reply]
If you go to the `code` tab of this GitHub and go to the `Foundry FundMe` section you will see a GitHub link to the codebase for the section, and there you can find the file you are looking for. That was how I found this https://github.com/Cyfrin/foundry-fund-me-cu/blob/main/test/unit/ZkSyncDevOps.t.sol
